### DISCUSSION TITLE
Awesome list for Octoherd scripts

### DISCUSSION THREAD

**Discussion by robvanderleek** [octoherd/octoherd#30] [General]
I've created the first revision of an Awesome list for Octoherd scripts: https://github.com/robvanderleek/awesome-octoherd-scripts

Basically, (almost) all the scripts tagged with `octoherd-script` I could find. Please let me know if I forgot scripts or included scripts that are not for general use.

Curious to hear your feedback. Do you think such a list is useful? Should the scripts be categorized? Suggestions are very welcome!

FD: I'm the author of the Repo Meister tool in the list.

**Comment by gr2m** [comment]
Nice! Thanks for sharing! Maybe tweet about it and mention @octoherd, then we can retweet it?

**Reply by robvanderleek** [reply]
I've tweeted about the list: https://twitter.com/robvanderleek/status/1488654236082712580?s=20&t=FukOMxILaGgihjPzHLpVEg
Hope it's OK that I've used the Octocat GIF.
(just see the like and reply, thank you!)
